integrations/transportation/tfl.js

```diff
@@ -124,9 +124,28 @@ async function collect(state) {
       throw new HTTPError(e);
     });
 
+  // Update contactless cards
+  const contactlessCardResponse = await fetch(`${BASE_PATH}/Contactless/Cards`, {
+    method: 'get',
+    headers: {
+      'x-zumo-auth': newState.accessToken,
+    },
+  })
+    .then(response => response.json())
+    .catch((e) => {
+      throw new HTTPError(e);
+    });
+
   const oysterCards = oysterCardResponse.OysterCards;
+  const contactlessCards = contactlessCardResponse;
+
+  newState.oysterCardNumbers = oysterCards && oysterCards.length > 0
+    ? oysterCards.map(oc => oc.OysterCardNumber)
+    : [];
 
-  newState.oysterCardNumbers = oysterCards.map(oc => oc.OysterCardNumber);
+  newState.contactlessCardIds = contactlessCards && contactlessCards.length > 0
+    ? contactlessCards.map(cc => cc.Id)
+    : [];
```

Comment:
maybe worth adding a `logDebug` where you log the number of cards of each type that have been detected?

Reply:
Sounds good, have added